readme.md

```diff
@@ -34,6 +39,16 @@ Type: `number` `string`
 
 Process ID or name to check.
 
+### processExists.array(input)
+
+Returns a `Promise<Array>` with the processes that exists.
```

Comment:
Wonder whether it should return this, an array of booleans, or a Map of input:result. I’m leaning towards the latter. Thoughts?

Reply:
If we add the latter, we could add an additional method called `filterExists` For what you have now.

Reply:
An array of booleans doesn't sound too useful? You wouldn't know what which one would be. I like the latter though!

Reply:
> An array of booleans doesn't sound too useful?

Well, you could match it on the index, but I agree, it's the least useful of the options. Let's go with the latter.

Reply:
> Well, you could match it on the index

Dunno why I thought that wouldn't work. But yes let's go with `Map`.